policies/approval.yml

```diff
@@ -52,21 +52,10 @@ approval_rules:
         # will be accepted as approval candidates. Default is true.
         github_review: true
 
-  # TODO: check who has permissions to add labels before enabling
-  - name: pull request has the no-review label
+  - name: renovate pull request
     if:
-      # "author_is_only_contributor", when true, is satisfied if all commits in the
-      # pull request are authored by and committed by the user who opened the pull
-      # request. When false, it is satisfied if at least one commit in the pull
-      # request was authored or committed by another user.
-      author_is_only_contributor: false
-
       # "has_labels" is satisfied if the pull request has the specified labels
       # applied
       has_labels:
-        - "no-review"
-
-    requires:
-      # "count" is the number of required approvals. The default is 0, meaning no
-      # approval is necessary.
-      count: 0 # indicates "auto-approval"
```

Comment:
I think we need to keep `count: 0` in order to auto-approve with these labels?

Reply:
it's the default, do you want me still write it out?

Reply:
Haha, is it really? No need to write it out, I know how you feel about defaults!